docs/assets/css/custom.css

```diff
@@ -257,9 +257,14 @@ li.folder > ul > li.file {
 .sidebar .search input[type="search"],
 .sidebar .search input[type="search"]:focus {
   background-color: #2d1763;
+  background-image: url("data:image/svg+xml,%3Csvg height='20px' width='20px' viewBox='0 0 24 24' fill='none' stroke='rgba(255, 255, 255, 0.5)' stroke-width='1.5' stroke-linecap='round' stroke-linejoin='round' preserveAspectRatio='xMidYMid meet' xmlns='http://www.w3.org/2000/svg'%3E%3Ccircle cx='10.5' cy='10.5' r='7.5' vector-effect='non-scaling-stroke'%3E%3C/circle%3E%3Cline x1='21' y1='21' x2='15.8' y2='15.8' vector-effect='non-scaling-stroke'%3E%3C/line%3E%3C/svg%3E");
```

Comment:
Are you trying to override the SVG from `plugin-search`?

![image](https://github.com/eth-protocol-fellows/protocol-studies/assets/10168946/7e58c967-515d-4883-a751-eb1a71105762)

Reply:
thank you for reviewing,


> Thanks for your contribution. Could you describe what issue are you fixing?
> 

I was trying to fix the placeholder and icon within the search input, because it is blurred, not contrast with the background (_hard to see_).
> Are you trying to override the SVG from `plugin-search`?
> 
ohh, is that cause by the plugin? I don't know about that

Reply:
+1 to rahul's question. The current search bar looks good to me, not sure what the issue. Can you please send a screenshot and describe it?

Reply:
<img width="433" alt="Screenshot 2024-06-10 at 8 45 21 in the morning" src="https://github.com/eth-protocol-fellows/protocol-studies/assets/52003343/adbed985-2474-4fe0-a41c-6eeca46d95cd">

I think it is caused by the theme. it will show like above if in light theme

Reply:
<img width="998" alt="Screenshot 2024-06-10 at 8 52 25 in the morning" src="https://github.com/eth-protocol-fellows/protocol-studies/assets/52003343/08086155-5db7-4892-9a7e-4bd9a0eacc7a">

the code block is also having contrast issue with the background when in light theme

Reply:
Interesting, this is how ti renders for me: 
![image](https://github.com/eth-protocol-fellows/protocol-studies/assets/61149543/e7fa77e1-3166-4bea-967e-ae0c19a53f6e)

Can you report what browser/OS are you using? Are you sure it's not caused by extension or some settings? 
Across all my browsers and devices, both search bar placeholder and block of code are readable. But I agree it should be more explicit in css to avoid breaking in some browsers

Reply:
yeah, it's working fine in dark theme. I use Brave browser on MacOS computer